packages/leancode_lint/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## 1.3.1
```

Comment:
Defo 1.4.0 because of the unawaited removal

Reply:
Actually, it should be 2.0.0 even

Reply:
thanks, I was inclined to do this as well but not sure